src/tests/Microsoft.PowerFx.Interpreter.Tests/RecalcEngineTests.cs

```diff
@@ -588,29 +588,34 @@ private void CheckThroughInterface(IPowerFxEngine engine)
         [Fact]
         public void RecalcEngineLocksConfig()

```

Comment:
>RecalcEngineLocksConfig [](http://example.com/codeflow?start=20&length=23)

If we can mutate the config (by adding new functions / symbols), then we should rename this :-)

Reply:
+1. Will rename. We no longer lock config.